linode/manager
# feat: [M3-5561] - Add Private IP checkbox when cloning a Linode

## Description 📝
Added Private IP checkbox when cloning a Linode.
Unit test coverage for `AddonsPanel`



## Preview 📷
**Include a screenshot or screen recording of the change**

| Before  | After   |
|------|---------|
| ![image](https://user-images.githubusercontent.com/119517080/233516275-4b8a1591-1ae0-4a50-9987-44c9cf12eb52.png) | ![image](https://user-images.githubusercontent.com/119517080/233516375-5cd4325b-9223-4e72-ac67-e46e245dce82.png) |

## How to test 🧪
- Navigate to [create Linode ](http://localhost:3000/linodes/create)
- Create Linode make sure to check private_ip address.
- Clone the Linode, Validate private_ip address checkbox.
- The checkbox should default to being checked if the source Linode has a Private IP allocated to it.
- yarn test
- yarn cy:run
